x-pack/plugins/gis/public/shared/layers/layer.js

```diff
@@ -136,6 +142,19 @@ export class ALayer {
     return this._source.renderSourceSettingsEditor({ onChange });
   };
 
+  renderTocEntryIcon({ fitToBounds, zoom, toggleVisible }) {
+    return (
+      <LayerTocActions
+        layer={this}
+        fitToBounds={fitToBounds}
```

Comment:
`fitToBounds` and `toggleVisible` are inconsistent. `fitToBounds` is a wrapper to `fitToBounds(layerId)` created in toc_entry/view.js while toggleVisible is just the original function provided by the redux connector. `toggleVisible` should use the same pattern